client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -66,28 +66,39 @@ class StatsPeriodNavigation extends PureComponent {
 		const nextDayQuery = qs.stringify( Object.assign( {}, queryParams, { startDate: nextDay } ), {
 			addQueryPrefix: true,
 		} );
+		const isNextDisabled = hideNextArrow || isToday;
 
 		return (
 			<div className="stats-period-navigation">
 				<div className="stats-period-navigation__children">{ children }</div>
-				<a
-					className={ classNames( 'stats-period-navigation__previous', {
-						'is-disabled': hidePreviousArrow,
-					} ) }
-					href={ `${ url }${ previousDayQuery }` }
-					onClick={ this.handleClickPrevious }
-				>
-					<Icon className="gridicon" icon={ arrowLeft } />
-				</a>
-				<a
-					className={ classNames( 'stats-period-navigation__next', {
-						'is-disabled': hideNextArrow || isToday,
-					} ) }
-					href={ `${ url }${ nextDayQuery }` }
-					onClick={ this.handleClickNext }
-				>
-					<Icon className="gridicon" icon={ arrowRight } />
-				</a>
+
+				{ ! hidePreviousArrow ? (
```

Comment:
Discussion about the HTML structure [started here](https://github.com/Automattic/wp-calypso/pull/71159/files#r1051282421).

Reply:
The comment doesn't apply anymore after I simplified the controller and used a new navigation component. So I'll revert this file to its original state.